raiden/tests/unit/test_upgrade.py

```diff
@@ -135,8 +135,6 @@ def test_upgrade_manager_restores_backup(tmp_path, monkeypatch):
         {'_type': 'raiden.transfer.state_change.ActionInitChain'},
     )
     assert state_change_record.data is not None
-    assert not old_db_filename.exists()
```

Comment:
This PR changes the semantics of the upgrade manager and the files are not renamed or deleted anymore